erpnext-openmrs/src/main/java/com/ozonehis/eip/erpnext/openmrs/handlers/ItemHandler.java

```diff
@@ -38,25 +37,24 @@ public class ItemHandler {
     @Autowired
     private QuotationItemMapper<Resource> quotationItemMapper;
 
-    public Optional<Item> itemExists(String name) {
+    public Optional<Item> getItem(String name) {
         try (FrappeResponse response = frappeClient.get("Item", name).execute()) {
-            if (response.code() == HttpStatus.SC_NOT_FOUND) {
-                return Optional.empty();
-            } else if (response.code() == HttpStatus.SC_OK) {
-                TypeReference<FrappeSingularDataWrapper<Item>> typeReference = new TypeReference<>() {};
-
-                return Optional.ofNullable(response.returnAs(typeReference).getData());
-            } else {
-                log.warn("Item with name {} does not exist", name);
-                return Optional.empty();
-            }
+            return switch (response.code()) {
+                case HttpStatus.SC_NOT_FOUND -> Optional.empty();
+                case HttpStatus.SC_OK -> {
+                    TypeReference<FrappeSingularDataWrapper<Item>> typeReference = new TypeReference<>() {};
+                    yield Optional.ofNullable(response.returnAs(typeReference).getData());
```

Comment:
We should stick to one coding style. Let's separate `switch` statement from `return` statement

Reply:
This is the Java 17 switch syntax. I like it, it's concise & expressive.